### PR TITLE
Use single quote.

### PR DESCRIPTION
**Pull request by drupol**
Use single quote everywhere.

### CODE REVIEW

**Comment by drupol** [comment]
@cweagans Could you review this ?

**Comment by cweagans** [comment]
> Use single quote everywhere.

Why?

**Comment by drupol** [comment]
We usually use single quote when using simple strings. For more complex strings with variables, we should use double quote.

See: http://php.net/language.types.string#language.types.string.syntax.single
See: https://phptherightway.com/pages/The-Basics.html

Using single quotes is therefore faster (_I agree this is micro-optimization_).

**Comment by drupol** [comment]
I'm going to rebase this now.

**Comment by cweagans** [comment]
I'd rather not do this, mostly because this style probably won't be maintained as development progresses.

**Comment by drupol** [comment]
Oh sad... Let me try again.

We could enforce it through multiple ways:

# With PHP-CS-Fixer: The Symfony coding standard uses it everywhere through PHP-CS-Fixer, see: https://github.com/FriendsOfPHP/PHP-CS-Fixer (`CTRL+F single_quote`).
# With .editorconfig: A very simple file at the root of the project with a couple of configuration options, Drupal 8 is using it as well and there's a directive to configure the quote type (https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties). When using the .editorconfig plugin on your favorite IDE, it will configure it thanks to that file. It's very handy.
# Drupal is using it by default: https://www.drupal.org/docs/develop/standards/coding-standards#quotes

I think the lightest way to enforce it would be to use .editorconfig file.
